packages/taikoon-ui/src/components/sections/Footer.section.svelte

```diff
@@ -1,261 +1,17 @@
 <script lang="ts">
-  import { json, t } from 'svelte-i18n';
+  import { Footer } from 'ui-lib';
```

Comment:
Let's not change multiple packages in a single PR, even though they are related. 

Split this PR into 3 to keep everything organized.

1. ui-lib repo setup (workflow, pr title)
2. ui-lib package add
3. taikoon package changes